interview-questions.md

```diff
@@ -16,44 +16,44 @@ Researched answer:
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: A git is the data and what it is contained in. Github is a place or cloud for the data to be stored making it easier to be shared and managed.
 
-Researched answer:
+Researched answer: Git is a free open-sourced version control system widely used in the programming world. It allows people to contribute to the same code. It was created to track changes in code. GitHub is the hosting platform for the Git. 
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: Booleans are true or false statements. Logical operators && (and) and || (or) will return a boolean value. 
 
-Researched answer:
+Researched answer: Equality operators compare one value to the other and judge if the values equate to the same thing. If they do equate to the same a Boolean value of true is returned if not we get a Boolean value of false.
```

Comment:
This question is asking which operators will return a Boolean. For example, if we did `console.log(5 > 3)`, it will return the Boolean of `true`, which means the relational operator `>` will return a Boolean. The question is asking for all operators that will also return a Boolean.